postgres/init.sls

```diff
@@ -8,6 +8,11 @@ include:
 {{ postgres.conf_dir }}:
   file.directory:
     - makedirs: True
+    - user: postgres
+    - group: postgres
+    - dir_mode: 700
+    - clean: True
```

Comment:
@blbradley I think that it's possible to put formulas inside Salt sources and then use this: https://docs.saltstack.com/en/latest/topics/development/tests/unit.html

Thank you for kitchen.ci, I'll definitely check it out.